funcx_sdk/parsl/app/errors.py

```diff
@@ -1,54 +0,0 @@
-import logging
```

Comment:
Is the removal of this file somehow essential to this PR?

Can it be done in a separate PR which we could merge immediately, and then this work rebased?